AElf.Sdk.CSharp.Tests/TestSdkCSharpAElfModule.cs

```diff
@@ -1,26 +1,44 @@
-﻿using AElf.Database;
+﻿using System.Threading.Tasks;
+using AElf.Common;
+using AElf.Database;
+using AElf.Kernel;
 using AElf.Kernel.Infrastructure;
-using AElf.Kernel.SmartContract;
+using AElf.Kernel.SmartContract.Application;
 using AElf.Kernel.SmartContract.Contexts;
 using AElf.Modularity;
 using AElf.TestBase;
 using Microsoft.Extensions.DependencyInjection;
+using Moq;
 using Volo.Abp.Modularity;
 
 namespace AElf.Sdk.CSharp.Tests
 {
     [DependsOn(
-        typeof(DatabaseAElfModule),
         typeof(TestBaseAElfModule),
-        typeof(SmartContractAElfModule))]
-    public class TestSdkCSharpAElfModule: AElfModule
+        typeof(TestBaseKernelAElfModule))]
```

Comment:
should depend on AElf.Sdk.CSharp.CSharpSdkModule

Reply:
AElf.Sdk.CSharp doesn't have his own module class.